beacon-chain/core/helpers/committee.go

```diff
@@ -203,20 +203,17 @@ func CrosslinkCommitteesAtSlot(
 func Shuffling(
 	seed [32]byte,
 	validators []*pb.Validator,
-	slot uint64) ([][]uint64, error) {
-
-	// Normalize slot to start of epoch boundary.
-	slot -= slot % params.BeaconConfig().SlotsPerEpoch
+	epoch uint64) ([][]uint64, error) {
 
 	// Figure out how many committees can be in a single slot.
-	activeIndices := ActiveValidatorIndices(validators, slot)
+	activeIndices := ActiveValidatorIndices(validators, epoch)
```

Comment:
2.) if we use shuffling epoch here, unless there's a new shuffling epoch in state right away, we won't be getting the latest validators (newly registered)

Reply:
3.) we should be using epoch instead of slot